skimage/filters/_gaussian.py

```diff
@@ -10,7 +10,7 @@
 
 
 def gaussian(image, sigma=1, output=None, mode='nearest', cval=0,
-             multichannel=None, preserve_range=False, truncate=4.0):
+             multichannel=None, preserve_range=True, truncate=4.0):
```

Comment:
I quite didn't see point of doing that here. See part of current docstring:

>     preserve_range : bool, optional
        Whether to keep the original range of values. Otherwise, the input
        image is converted according to the conventions of ``img_as_float``.
        Also see
        https://scikit-image.org/docs/dev/user_guide/data_types.html
        
 But maybe it should be rewritten to be more specific. If you consider that as necessery - I may do that